app/admin/concepts/statistic/operation.rb

```diff
@@ -0,0 +1,45 @@
+class StatisticOperation
+  class EventCount
+    class Show
+      def call(params = {})
+        Event.count
+      end
+    end
+  end
+
+  class TotalParticipants
```

Comment:
Just to make sure: are you on purpose not distinguishing between `pending` and `published` Events here?

Don't know to which extend this *feature* is actually used though.